addons/mail/static/src/core/common/message.xml

```diff
@@ -73,7 +73,7 @@
                                                     'o-green': message.bubbleColor === 'green',
                                                     'o-orange': message.bubbleColor === 'orange',
                                                     }" t-attf-class="{{ isAlignedRight ? 'rounded-start-3' : 'rounded-end-3' }}"/>
-                                                <MessageInReply t-if="message.parentMessage" message="message" onClick="props.onParentMessageClick"/>
+                                                <MessageInReply t-if="message.parentMessage and !message.isEmpty" message="message" onClick="props.onParentMessageClick"/>
```

Comment:
I don't think we want to just hide it in the interface, what we probably want to do is to actually remove the `parent_id` when deleting a message in a channel.